docs/src/roadmap.md

```diff
@@ -38,9 +38,9 @@ Some but not all of these tasks are summarized in the
     with JuMP's documentation.
 - Developer experience
   - Get JuMP’s unit tests running in less than two minutes. See [#1745](https://github.com/jump-dev/JuMP.jl/pull/1745).
-- All solvers should complete the transition to MOI.
+- All solvers should complete the transition to MOI. (**Done**)
 - Provide packages for installing Bonmin and Couenne.
-- [MathOptFormat](https://github.com/odow/MathOptFormat.jl) 1.0
+- [MathOptFormat](https://github.com/odow/MathOptFormat.jl) 1.0 (**Done**)
```

Comment:
Claiming this is done since it is integrated into MOI, and we can read/write from JuMP. Future changes will not be user-facing, or will be addition of new sets.